django/bosscore/test/test_group_views.py

```diff
@@ -507,6 +525,20 @@ def test_remove_maintainer_group(self):
         self.assertEqual(response.status_code, 200)
         self.assertIn('testuser', response.data['maintainers'])
 
+    def test_remove_bossadmin_maintainer_group_fails(self):
+        """ Test removal of bossadmin as a maintainer of a group fails"""
+
+        # Check if bossadmin user is a member of the group
+        url = '/' + version + '/groups/unittest/maintainers/bossadmin/'
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data['result'], True)
+
+        # Attempt removal of bossadmin from the group
+        url = '/' + version + '/groups/unittest/maintainers/bossadmin/'
+        response = self.client.delete(url)
+        self.assertRaises(Exception)
```

Comment:
Does this test pass? I've only used this like so:
```python
with self.assertRaises(Exception):
    response = self.client.delete(url)
```

Reply:
It does pass, but maybe I should try it in your format? 
the unittest module seems to notice there is an exception raised even if the assertion is after the line where the exception is raised. Is this ok @derek-pryor ?

Reply:
Actually just revised, this was a very good catch. The test was passing, but it was passing incorrectly. Indeed the response will actually be a 404 not a raised Exception. So I changed the code entirely to be:
```
response = self.client.delete(url)
self.assertEqual(response.status_code, 400)
```
Attempt to delete bossadmin returns an  ErrorCodes.BAD_REQUEST which is a 400 not an exception. Thanks Tim!

Pushing change now